frontend/app/src/components/status/BalanceSavedIndicator.vue

```diff
@@ -7,11 +7,15 @@
     z-index="215"
   >
     <template #activator="{ on }">
-      <v-btn icon class="secondary--text text--lighten-2" v-on="on">
+      <menu-tooltip-button
+        tooltip="Balance save"
```

Comment:
--> "Balance Save" (I'm not sure I like this one but I can't think of what else to call it). Any ideas?

Reply:
Balances Snapshot Save status. Or ... indeed Save Status for short.

Reply:
Balances Snapshot Save Status fit correctly to the tooltip so let's go for the explicit 👍